app/javascript/modCenter/singleArticle/index.jsx

```diff
@@ -31,12 +31,14 @@ export default class SingleArticle extends Component {
     const { articleOpened } = this.state;
     const { id, title, publishedAt, cachedTagList, user, key } = this.props;
     const tags = cachedTagList.split(', ').map((tag) => {
-      return (
-        <span className="mod-article-tag fs-s ff-accent lh-base" key={key}>
-          <span className="article-hash-tag">#</span>
-          {tag}
-        </span>
-      );
+      if (tag) {
+        return (
+          <span className="mod-article-tag fs-s ff-accent lh-base" key={key}>
+            <span className="article-hash-tag">#</span>
+            {tag}
+          </span>
```

Comment:
We have this guy that could be reused:

```suggestion
          <span className="crayons-tag" key={key}>
            <span className="crayons-tag__prefix">#</span>
            {tag}
          </span>
```

Reply:
Ooh good catch, thanks! Btw we'll update this to use the Crayons table component in the future, maybe cooldown? 😄